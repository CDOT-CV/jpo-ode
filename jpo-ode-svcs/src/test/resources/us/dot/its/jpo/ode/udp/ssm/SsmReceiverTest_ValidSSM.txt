001e120000000005e9c04071a26614c06000040ba0