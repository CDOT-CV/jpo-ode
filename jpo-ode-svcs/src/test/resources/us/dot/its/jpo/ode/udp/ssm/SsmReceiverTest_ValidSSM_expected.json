{
  "metadata": {
    "recordType": "ssmTx",
    "securityResultCode": "success",
    "payloadType": "us.dot.its.jpo.ode.model.OdeAsn1Payload",
    "serialId": {
      "streamId": "db41bd96-51f6-40d7-a395-dee198042e5a",
      "bundleSize": 1,
      "bundleId": 0,
      "recordId": 0,
      "serialNumber": 0
    },
    "odeReceivedAt": "2024-11-26T23:53:21.120Z",
    "schemaVersion": 7,
    "maxDurationTime": 0,
    "recordGeneratedBy": "RSU",
    "sanitized": false,
    "asn1": "30303165313230303030303030303035653963303430373161323636313463303630303030343062613000",
    "originIp": "127.0.0.1",
    "ssmSource": "RSU"
  },
  "payload": {
    "dataType": "us.dot.its.jpo.ode.model.OdeHexByteArray",
    "data": {
      "bytes": "30303165313230303030303030303035653963303430373161323636313463303630303030343062613000"
    }
  }
}